SingularityService/src/main/java/com/hubspot/singularity/resources/DeployResource.java

```diff
@@ -50,6 +51,7 @@
 @Api(description="Manages Singularity Deploys for existing requests", value=DeployResource.PATH, position=2)
 public class DeployResource extends AbstractRequestResource {
   public static final String PATH = SingularityService.API_BASE_PATH + "/deploys";
+  public static final int DEFAULT_DEPLOY_STEP_WAIT_TIME_SECONDS = 60;
```

Comment:
@tpetr was debating making the default wait time 0, would that make more sense? i.e. move on to the next step as soon as the previous is finished 'rolling deploy', by default